recipes/einsums/meta.yaml

```diff
@@ -0,0 +1,107 @@
+{% set name = "Einsums" %}
+{% set version = "0.2" %}
+{% set sha256 = "6f0e209add31b38d58a557e14afce7b23d5dd684a70aaf15554469f19ce77d29" %}
+{% set build_number = 0 %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/{{ name }}/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: {{ build_number }}
+  string: {{ blas_impl }}_h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}
+  skip: true                                                               # [win]
+  binary_relocation: true
+  run_exports:
+    - {{ pin_subpackage('einsums', max_pin='x.x') }}
+  ignore_run_exports:
+    - elfutils                                                             # [linux]
+    - libzlib
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - cmake
+    - ninja
+    - llvm-openmp                                                          # [osx]
+    - libgomp                                                              # [linux]
+  host:
+    - catch2
+    - elfutils                                                             # [linux]
+    - fmt 10.1
+    - hdf5
+    - range-v3 0.12
+    - zlib
+    - llvm-openmp                                                          # [blas_impl == "mkl" and linux]
+    - mkl-devel {{ mkl }}                                                  # [blas_impl == "mkl"]
+    - blas-devel * *_{{ blas_impl }}                                       # [blas_impl == "openblas"]
+    - fftw                                                                 # [blas_impl == "openblas"]
+    - openblas * openmp_*                                                  # [blas_impl == "openblas"]
+  run:
+    - llvm-openmp                                                          # [blas_impl == "mkl" and linux]
+    - libopenblas * openmp_*                                               # [blas_impl == "openblas"]
+
+test:
+  requires:
+    - {{ compiler('cxx') }}
+    - cmake
+    - ninja
+    - range-v3 0.12
+    - zlib
+    - mkl-devel                                                            # [blas_impl == "mkl"]
+    - blas-devel * *_openblas                                              # [blas_impl == "openblas"]
+  files:
+    - timing/CMakeLists.txt
+  source_files:
+    - timing/main.cpp
+  commands:
+    # Verify library
+    - test -f $PREFIX/lib/libeinsums$SHLIB_EXT                             # [unix]
```

Comment:
```suggestion
    # Verify library
    - test -f $PREFIX/lib/libeinsums.so.{{ version }}                      # [linux]
    - test -f $PREFIX/lib/libeinsums$SHLIB_EXT                             # [unix]
```

Reply:
My only suggestion is to start checking the explicit so version to prevent ABI breaks in the future.

Reply:
I'll add in the SO check. Thanks for the merge!